controller/cmd/proxy-injector/main.go

```diff
@@ -1,16 +1,18 @@
-package main
+package proxyinjector
 
 import (
 	"github.com/linkerd/linkerd2/controller/k8s"
 	injector "github.com/linkerd/linkerd2/controller/proxy-injector"
 	"github.com/linkerd/linkerd2/controller/webhook"
 )
 
-func main() {
+// Main executes the proxy-injector subcommand
+func Main(args []string) {
 	webhook.Launch(
 		[]k8s.APIResource{k8s.NS, k8s.Deploy, k8s.RC, k8s.RS, k8s.Job, k8s.DS, k8s.SS, k8s.Pod},
 		9995,
 		injector.Inject,
-		"linkerd-proxy-injector",
+		"proxy-injector",
```

Comment:
@alpeb @ihcsim I modified both webhook component names so they nicely match the subcommand names. All tests are passing, but I wasn't totally sure if changing these strings had other implications. 🤷‍♂

Reply:
That label is used when an injection event is raised, to identify the source of the event:
```bash
$ kubectl -n emojivoto get events -owide
LAST SEEN   TYPE     REASON              OBJECT                        SUBOBJECT                           SOURCE                   MESSAGE                                                                            FIRST SEEN   COUNT   NAME
38s         Normal   Injected            deployment/emoji                                                  linkerd-proxy-injector   Linkerd sidecar proxy injected                                                     38s          1       emoji.15c156b292ecdb8f
```
I'd leave the full "linkerd-proxy-injector" label to fully identify the event, even though Linkerd is also mentioned in the message...